tests/pass/shims/time-with-isolation.rs

```diff
@@ -26,7 +20,9 @@ fn main() {
     let diff = now2.duration_since(now1);
     assert_eq!(now1 + diff, now2);
     assert_eq!(now2 - diff, now1);
-    duration_sanity(diff);
+    // The virtual clock is deterministic and I got 29us. This should be updated if
+    // `NANOSECONDS_PER_BASIC_BLOCK` changes.
+    assert_eq!(diff.as_micros(), 29);
```

Comment:
Might be good to add some slack; this depends on exactly how many basic blocks there are in `drop(vec![42])`, which can change as rustc changes.